src/golang/lib/models/shared/artifact_type.go

```diff
@@ -16,6 +16,20 @@ const (
 	PicklableArtifact ArtifactType = "picklable"
 )
 
+var ArtifactTypes = []ArtifactType{
+	UntypedArtifact,
```

Comment:
I feel like we should avoid defining this list if its just used for testing and instead define it in the testing package.